packages/next/build/webpack-config.ts

```diff
@@ -179,6 +179,100 @@ export default async function getBaseWebpackConfig(
 
   const devtool = dev || debug ? 'cheap-module-source-map' : false
 
+  // Contains various versions of the Webpack SplitChunksPlugin used in different build types
+  const splitChunksConfigs: {
+    [propName: string]: webpack.Options.SplitChunksOptions
+  } = {
+    dev: {
+      cacheGroups: {
+        default: false,
+        vendors: false,
+      },
+    },
+    selective: {
+      cacheGroups: {
+        default: false,
+        vendors: false,
+        react: {
+          name: 'commons',
+          chunks: 'all',
+          test: /[\\/]node_modules[\\/](react|react-dom)[\\/]/,
+        },
+      },
+    },
+    prod: {
+      chunks: 'all',
+      cacheGroups: {
+        default: false,
+        vendors: false,
+        commons: {
+          name: 'commons',
+          chunks: 'all',
+          minChunks: totalPages > 2 ? totalPages * 0.5 : 2,
+        },
+        react: {
+          name: 'commons',
+          chunks: 'all',
+          test: /[\\/]node_modules[\\/](react|react-dom)[\\/]/,
```

Comment:
want to add the `scheduler` package to this whitelist while we're editing? @timneutkens had this in his experimental branch.

Reply:
I assume this will be flagged 100% of the time as it's a dep of react. But doesn't hurt to add it I think 🤔

Reply:
We've for some reason seen cases where it ends up in a number of route chunks. I actually don't have a clear answer why though.

Reply:
Might also be a good idea to detect multiple React versions, sounds out of scope for this particular work, but have seen that happen quite a few times too.

Reply:
I added the scheduler module to the framework cacheGrou